Makefile

```diff
@@ -7,7 +7,7 @@ NPM_FLAGS     := --prefer-offline
 DOCKER_FLAGS  := --interactive --tty
 TEST_DIR      :=
 REDUX_LOG     :=
-UNIQUE_APP_CONTAINER := $(shell uuidgen)-app
```

Comment:
tbh, I'm creeped out that simple assignment `:=` doesn't work here and I don't understand why not, but it seems that the MAKEFLAGS aren't available at this point in the script.

So-called recursive assignment is annoying because it causes a new UUID to be generated for each step, but it might be harmless.